staging/src/k8s.io/kubectl/pkg/describe/describe.go

```diff
@@ -5451,10 +5451,13 @@ func formatEndpointSlices(endpointSlices []discoveryv1.EndpointSlice, ports sets
 				if len(list) == max {
 					more = true
 				}
-				if !more {
-					list = append(list, endpointSlices[i].Endpoints[j].Addresses[0])
+				isReady := endpointSlices[i].Endpoints[j].Conditions.Ready
```

Comment:
```suggestion
		// ready indicates that this endpoint is prepared to receive traffic,
		// according to whatever system is managing the endpoint. A nil value
		// indicates an unknown state. In most cases consumers should interpret this
		// unknown state as ready.
		// More info: vendor/k8s.io/api/discovery/v1/types.go
		isReady := endpointSlices[i].Endpoints[j].Conditions.Ready == nil || *endpointSlices[i].Endpoints[j].Conditions.Ready
```

Reply:
> endpointSlices[i].Endpoints[j].Conditions.Ready == nil || *endpointSlices[i].Endpoints[j].Conditions.Ready

it should be `endpointSlices[i].Endpoints[j].Conditions.Ready == nil && *endpointSlices[i].Endpoints[j].Conditions.Ready`？

Reply:
>  // A nil value indicates an unknown state. In most cases consumers should interpret this
    // unknown state as ready.

This is a little weird for me.  Thank you very much @aojea